### PR TITLE
Use Symfony LTS 4.4

### PR DESCRIPTION
**Pull request by cybernet**
https://symfony.com/releases/4.4

 End of bug fixes: November 2022

### CODE REVIEW

**Comment by BoShurik** [comment]
Thanks for contribution
You forgot to change lock-file, as it's a project, not a library.
Also, I prefer latest (5.0.0) version of symfony

**Comment by cybernet** [comment]
LTS is better, in my opinion

**Comment by BoShurik** [comment]
Why? It's just an example, and is better to keep it up to date. 
People will refer to it when they want to create new project and it likely be symfony 5

**Comment by cybernet** [comment]
3.4 is still being used because it's LTS, even some use 2.8 for the same reason

File: composer.json [hunk 1]
```diff
@@ -7,14 +7,14 @@
         "ext-iconv": "*",
         "boshurik/telegram-bot-bundle": "^3.1",
```
Comment (BoShurik) [inline composer.json:8]: Please add ^4.0 version of the bundle

File: composer.json [hunk 2]
```diff
@@ -7,14 +7,14 @@
...
         "sensio/framework-extra-bundle": "^5.3",
-        "symfony/console": "4.2.*",
-        "symfony/dotenv": "4.2.*",
+        "symfony/console": "4.4.*",
+        "symfony/dotenv": "4.4.*",
         "symfony/flex": "^1.1",
         "symfony/framework-bundle": "4.2.*",
```
Comment (BoShurik) [inline composer.json:13]: Must be `4.4.*`

**Review by BoShurik — CHANGES_REQUESTED** [review]

**Comment by cybernet** [comment]
> 
> 
> Thanks for contribution
> You forgot to change lock-file, as it's a project, not a library.
> Also, I prefer latest (5.0.0) version of symfony

Then you should also use EventSubscriberInterface instead of Event :)

**Comment by BoShurik** [comment]
> Then you should also use EventSubscriberInterface instead of Event :)

Do you mean changed signature of EventSubscriberInterface? Yes, it's not just update version constraints. It would be great to fix all deprecations in 4.4

**Comment by BoShurik** [comment]
Updates to 5.1 in 4a7a61d

**Pull request closed without merging**